From 7373af0347e0838a6865a6e575496a851e8f4cff Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Wed, 23 Sep 2015 09:23:43 -0500
Subject: [PATCH] MLK-11611: fix build error when use mfg_defconfig

fix build error:
arch/arm/mach-imx/ddr3_freq_imx7d.S:425: Error: selected processor does not support ARM mode `dsb'

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 arch/arm/configs/imx_v7_mfg_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_mfg_defconfig b/arch/arm/configs/imx_v7_mfg_defconfig
index dbe140a..3093eec 100644
--- a/arch/arm/configs/imx_v7_mfg_defconfig
+++ b/arch/arm/configs/imx_v7_mfg_defconfig
@@ -17,7 +17,6 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_MODVERSIONS=y
 CONFIG_MODULE_SRCVERSION_ALL=y
 # CONFIG_BLK_DEV_BSG is not set
-CONFIG_ARCH_MULTI_V6=y
 CONFIG_GPIO_PCA953X=y
 CONFIG_ARCH_MXC=y
 # CONFIG_MACH_MX31ADS is not set
-- 
1.8.0

